tests/lib/prepare-restore.sh

```diff
@@ -294,6 +294,11 @@ prepare_project() {
     RateLimitBurst=0
 EOF
     systemctl restart systemd-journald.service
+
+    # Make systemd able to run avoiding to recompile SELinux policy sources.
+    if [[ "$SPREAD_SYSTEM" = fedora-* ]] && [ "$SPREAD_BACKEND" = google ]; then
+        semanage permissive -a init_t || true
```

Comment:
Can you please add a todo here. This will mask any errors that are caused by bugs in our SElinux policy.